Extensions/Emails/Services/EmailService.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Generic;
+using System.Linq;
+using Lombiq.HelpfulExtensions.Extensions.Emails.Models;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using OrchardCore.Email;
+using OrchardCore.Environment.Shell.Scope;
+
+namespace Lombiq.HelpfulExtensions.Extensions.Emails.Services
+{
+    public class EmailService : IEmailService
+    {
+        public void SendEmailDeferred(EmailParameters parameters) =>

```

Comment:
This service doesn't inject anything, so I wonder if it would be cleaner as an extension method for EmailParameters or ShellScope (ie `ShellScope.Current.SendEmailDeferred(parameters)`)?